README.md

```diff
@@ -27,6 +27,25 @@
 
 ## Getting Started ✨
 
+### Automated Installation
+For installing zap you can use our hacky [bash script](https://github.com/srevinsaju/zap/blob/main/install.sh).
```

Comment:
Ah right. Not everyone likes running bash scripts as root user lol.

Reply:
Agreed, I like the warning you mentioned about running bash scripts as root, but perhaps, it would be better if we promote local installation over the sudo part, but I can make that changes